nx.json

```diff
@@ -0,0 +1,29 @@
+{
+  "implicitDependencies": {
+    "package.json": "*"
+  },
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "nx/tasks-runners/default",
+      "options": {
+        "cacheableOperations": [
+          "test",
+          "build"
+        ],
+        "parallel": 5
+      }
+    }
+  },
+  "targetDefaults": {
+    "build": {
+      "dependsOn": [
+        "^build"
+      ]
+    }
+  },
```

Comment:
"The configuration above is identical to adding {"dependsOn": ["^build"]} to every build target of every project." https://nx.dev/configuration/packagejson#target-defaults

Reply:
Better: https://nx.dev/configuration/projectjson#dependson